package/firmware/phicommk3-firmware/Config.in

```diff
@@ -0,0 +1,18 @@
+choice
+    prompt "Select the wireless firmware"
+    default K3_WF_AC88U
+    depends on PACKAGE_phicommk3-firmware
+
+    config K3_WF_AC88U
+        bool "AC88U"
+
+    config K3_WF_EA9500
+        bool "EA9500"
+
+    config K3_WF_K3
+        bool "K3"
+        
+    config K3_WF_DHD24
+        bool "ASUS dhd24"
```

Comment:
Instead of making this a compile time option, I would prefer if you could create 4 opkg packages.
Why is the K3_WF_AC88U seletced as the default?

Reply:
Create 4 packages named like phicommk3-firmware-ea9500, then require one of theses package in the image makefile? I think I can test all of them and choose the best one to make just one package. What do you say?